Add tutorial: How to create unsubscribe surveys (fixes #11577)

Faisal-011 opened the pull request:
Closes #11577

## Changes
Added a new tutorial explaining how to create unsubscribe surveys using PostHog Surveys.
The tutorial includes prerequisites, step-by-step setup, trigger configuration, and analysis.

## Checklist
- [x] Words are spelled using American English
- [x] Titles are in sentence case
- [x] Feature names use sentence case too ("product analytics" not "Product Analytics")
- [x] Used relative URLs for internal links
- [x] The template has been filled out completely

## Article checklist
- [x] Added 3–5 internal links (to /docs/surveys, /docs/insights, /signup)
- [x] Checked the preview build of the article
- [x] Date is set to today's date in the frontmatter
- [x] Added to the relevant "Tutorials" and "Guides" section if applicable

Faisal-011 committed:
Created tutorial: How to create unsubscribe surveys

Faisal-011 committed:
Updated tutorial format: How to create unsubscribe surveys

Faisal-011 commented:
Thanks for the feedback!

I’ve updated the tutorial to match PostHog’s standard format and tone. The structure now follows the same layout as other tutorials including clear steps, headings, example queries, and placeholder screenshots.

Note: I’ve used placeholder screenshot links to show where visuals would appear. I can update them later with real screenshots if needed.

Please let me know if any further adjustments or formatting tweaks are needed.

smallbrownbike committed:
Merge branch 'master' into patch-3

Faisal-011 committed:
Merge branch 'master' into patch-3

ivanagas commented on contents/tutorialsAdd tutorial: How to create unsubscribe surveys:
```diff
@@ -0,0 +1,163 @@
+7. Click **Next** to continue.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/v1729000001/unsubscribe-survey-create.png"
```
Could you post these images in GitHub and I will upload them to the CRM?

ivanagas commented on contents/tutorialsAdd tutorial: How to create unsubscribe surveys:
can you fix the file name to be `unsubscribe-surveys.md`?

ivanagas commented on contents/tutorialsAdd tutorial: How to create unsubscribe surveys:
````diff
@@ -0,0 +1,163 @@
+  alt="Automating unsubscribe feedback in Zapier"
+  classes="rounded"
+/>
+```
````
```suggestion
<ProductScreenshot
  imageLight="https://res.cloudinary.com/demo/image/upload/v1729000000/unsubscribe-zapier.png"
  imageDark="https://res.cloudinary.com/demo/image/upload/v1729000000/unsubscribe-zapier-dark.png"
  alt="Automating unsubscribe feedback in Zapier"
  classes="rounded"
/>
```

ivanagas reviewed (changes requested):
some more recommendations!

Faisal-011 committed:
images for unsubscribe tutorial

Faisal-011 committed:
Add files via upload

Faisal-011 committed:
Merge branch 'master' into patch-3

Faisal-011 committed:
Revise unsubscribe surveys tutorial for UI updates

Updated the tutorial on creating unsubscribe surveys to reflect changes in PostHog's UI and terminology. Adjusted steps for clarity and accuracy.

Faisal-011 committed:
Merge branch 'master' into patch-3

Faisal-011 commented:
Follow-up: all requested changes applied. Let me know if anything else is needed.

ivanagas committed:
Merge branch 'master' into patch-3

ivanagas committed:
edits

ivanagas commented on contents/tutorials/unsubscribe-surveys.md:
```diff
@@ -0,0 +1,74 @@
+3. Back in your survey, scroll to the **Display conditions** section.
+4. Under **Users who match all of the following…**, add a **User sends events** filter for your `user_unsubscribed` action.
+5. Add a **URL contains** filter, such as `/unsubscribe`.
+6. Adjust any additional filters if needed (for example, only specific plans or countries).
```
Also, we should recommend users do everything they need to do for targeting before they create the survey so they don't need to go back to survey creation to sort targeting.

ivanagas commented on src/data/authors.json:
```diff
@@ -501,5 +501,12 @@
+        "name": "Faisal-011",
+        "role": "Contributor",
+        "link_type": "github",
+        "link_url": "https://github.com/Faisal-011"
```
Do you have a PostHog community profile? We can add the ID here: https://posthog.com/community
